macosx/it.lproj/Preferences.strings

```diff
@@ -217,6 +217,9 @@
 /* Class = "NSTextFieldCell"; title = "seconds"; ObjectID = "klQ-DW-Kc6"; */
 "klQ-DW-Kc6.title" = "secondi";
 
+/* Class = "NSButtonCell"; title = "Keep preset edits when selecting a new title"; ObjectID = "L3N-te-KLg"; */
+"L3N-te-KLg.title" = "Mantieni le modifiche delle preimpostazioni quando selezioni un nuovo titolo";
```

Comment:
I decided to keep the "preset" name in the Italian Mac translation, so this would be "Mantieni le modifiche del preset quando selezioni un nuovo titolo". I already updated it on Transifex.